script/lib/create-debian-package.js

```diff
@@ -226,13 +226,7 @@ module.exports = function(packagedAppPath) {
     `Copying generated package into "${outputDebianPackageFilePath}"`
   );
 
-  fs.chmodSync(
-    path.join(
-      debianPackageDirPath,
-      'chrome-sandbox'
-    ),
-    '4755'
-  );
+  fs.chmodSync(path.join(debianPackageDirPath, 'chrome-sandbox'), '4755');
```

Comment:
Thanks for reaching out!

Try moving this line to above line 220 to chmod the file _before_ it's added to the `.deb`.